blips/blip-0001.md

```diff
@@ -0,0 +1,289 @@
+```
+bLIP: 1
+Title: bLIP process
+Status: Active
+Author: Ryan Gentry <ryan@lightning.engineering>
+Created: 2021-05-21
+Post-History: 2021-06-30: https://lists.linuxfoundation.org/pipermail/lightning-dev/2021-June/003086.html
+  [lightning-dev] bLIPs: A proposal for community-driven app layer and protocol extension standardization
+License: CC0
+```
+
+# Abstract
+
+bLIP stands for Bitcoin Lightning Improvement Proposal. A bLIP is a design document 
+providing information to the Lightning community, or describing a new feature for 
+the Lightning Network. The bLIP should provide a concise technical specification of 
+the feature and a rationale for the feature. The bLIP author is responsible for 
+building consensus within the community and documenting dissenting opinions.
+
+# Copyright
+
+This bLIP is licensed under the CC0 license.
+
+# Rationale
+
+As the Lightning community has grown, new features, standards, and protocols have 
+been developed outside of the BOLT specification process: particularly at the 
+application level that isn’t described within the core BOLT documents. This is great! 
+But in the spirit of interoperability, documenting features, standards, and protocols 
+in a single location with a standard format will make it easy on future adopters. 
+
+In particular, there are three scarce sets of identifiers used in Lightning Network
+protocol development that benefit from central organization and documentation to
+avoid potential clashes:
+
+* **Feature Bits** are used to designate that a given node supports a given feature, and 
+are publicly broadcasted on the Lightning Network. Feature Bits are first introduced as
+optional with *odd* bits, and when upgraded to compulsory are switched to *even* bits, 
+so a new feature proposed in a bLIP should start with an *odd* bit. Feature Bits are 
+assigned and specified in [Bolt
+ #9](https://github.com/lightningnetwork/lightning-rfc/blob/master/09-features.md).
```

Comment:
That's not necessarily true. It's just that features can be either made `optional` or `mandatory` depending on what the node operator wants to achieve. I don't think we should have a requirement that proposals cannot use the `mandatory` bit, there are cases where it makes sense for them to do so.

Reply:
Makes sense. @TheBlueMatt's "if its intended to become universal or near universal, it must be a bolt" language from the #893 meeting better conveys what I was trying to here, so I will relax this language insert Matt's... somewhere :)